src/Kros.AspNetCore/Extensions/ConfigurationBuilderExtenstions.cs

```diff
@@ -53,6 +53,7 @@ public static IConfigurationBuilder AddLocalConfiguration(this IConfigurationBui
                 {
                     options
                         .Select($"{service}:*", hostingContext.HostingEnvironment.EnvironmentName)
+                        .Select($"{service}:*")
```

Comment:
I'm not sure if it's correct. According to the documentation to obtaining configuration without labelu you can use the following code `.Select(KeyFilter.Any, LabelFilter.Null)`

https://docs.microsoft.com/en-us/azure/azure-app-configuration/howto-labels-aspnet-core#loading-configuration-values-with-a-specified-label